modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/DefaultExecutionContext.java

```diff
@@ -181,8 +183,17 @@ public CoreEvent getEvent() {
   @Override
   public void changeEvent(CoreEvent updated) {
     requireNonNull(event);
-    event = updated;
     securityContext = event.getSecurityContext();
+    // Prevents possible lack of SdkContext if this context was generated by an interceptor (see MULE-19088)
```

Comment:
in the comment, point to the place in the code where the interceptor is creating the event without the context